libpd_wrapper/z_libpd.c

```diff
@@ -459,3 +545,32 @@ void libpd_set_polyaftertouchhook(const t_libpd_polyaftertouchhook hook) {
 void libpd_set_midibytehook(const t_libpd_midibytehook hook) {
   libpd_midibytehook = hook;
 }
+
+int libpd_startgui(char *path) {
+  int retval;
+  sys_lock();
+  retval = sys_startgui(path);
+  sys_unlock();
+  return retval;
+}
+
+void libpd_stopgui( void) {
+  sys_lock();
+  sys_stopgui();
```

Comment:
I'm getting a warning that this is undefined aka "Implicit declaration of function 'sys_stopgui' is invalid in C99." Maybe the declare is missing somewhere?